modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * registry/installing-openstack- .adoc
+// * registry/configuring-registry-operator.adoc
+// * registry/configuring-registry-storage-openstack-user-infrastructure.adoc
+
+[id="registry-configuring-registry-storage-swift-trust_{context}"]
+= Configuring the Image Registry Operator to trust Swift storage
+
+You must configure the Image Registry Operator to trust OpenStack Swift storage.
+
+////
+...to allow the client to pull the image layers from the image registry rather than from links directly from Swift.
+////
+
+.Procedure
+
+* To configure the Image Registry Operator to trust OpenStack Swift storage, change the ____ field on the `config.imageregistry` object to `spec.disableRedirect: true`.
```

Comment:
It's a draft, it should be a complete sentence.

Reply:
```suggestion
* To configure the Image Registry Operator to trust OpenStack Swift storage, in the `config.imageregistry` object, change the value of the  `spec.disableRedirect` field to `true`.
```